_pytest/config.py

```diff
@@ -236,6 +238,8 @@ def _try_load_conftest(self, anchor):
                     self._getconftestmodules(x)
 
     def _getconftestmodules(self, path):
+        if self._noconftest:
+            self._path2confmods[path] = []
```

Comment:
Makes sense - I did add it to `self._path2confmods` because i saw that being used in `_importconftest(...)`, but that's not called in this case anyways.